docs/workforce-management-framework/topics/pro-Using-Keycloak-implementation.adoc

```diff
@@ -0,0 +1,89 @@
+[id='enable-keycloak-integration-{chapter}']
+= Enable {WFM-RC-KeycloakName} Integration
+
+The {WFM-RC-NameLong} Demo Applications allow for Authentication and Access Management using
+link:http://www.keycloak.org/index.html[{WFM-RC-KeycloakName}].
+The {WFM-RC-NameLong} {WFM-RC-KeycloakName} integration is situated inside of the Demo Applications.
+
+By default, the {WFM-RC-KeycloakName} Authentication and Access Management is not setup with the Demo Applications.
+A {WFM-RC-NameLong} {WFM-RC-KeycloakName} Docker image has been created to allow easy deployment of a {WFM-RC-KeycloakName} server configured
+with the {WFM-RC-NameLong} demo data.
+
+== Run {WFM-RC-KeycloakName} Docker image
+
+The image is published to docker hub
+
+    docker run -p 8080:8080 feedhenry/raincatcher-keycloak
+
+For more information on this docker image see the docker image link:{WFM-RC-KeycloakDockerURL}{WFM-RC-Branch}/README.md[README.md].
+
+NOTE: If you wish to configure your own instance of {WFM-RC-KeycloakName} see the following
+link:{WFM-RC-KeycloakURL}getting_started/topics/first-boot.html[getting started guide]. You will need to import our Realm
+settings by importing link:{WFM-RC-KeycloakDockerURL}{WFM-RC-Branch}/data_files/raincatcher-realm.json[/data_files/raincatcher-realm.json].
+
+[[enabling-the-keycloak-demo-server]]
+== Enabling {WFM-RC-KeycloakName} on the Demo Server
+
+The following steps are for using the pre-configured docker image but will work for any instance of {WFM-RC-KeycloakName}
+provided you implement our Realm settings.
+
+- login into the {WFM-RC-KeycloakName} server via a browser using the following URL
+
+    http://localhost:8080/auth/
+
+- Click on `client` on the left menu
+- Select raincatcher-cloud from the list
+- Select `installation` from the selection bar at the top of the screen
+- From the `Format Options` drop down select `Keycloak OIDC JSON`
+- JSON Data similar to below will render (this data depends on the configuration of the {WFM-RC-KeycloakName} server and can vary)
+
+    {
+      "realm": "raincatcher",
```

Comment:
Updated config for the cloud app is.

{
    "realm": "raincatcher",
    "bearer-only": true,
    "auth-server-url": "http://localhost:8080/auth",
    "ssl-required": "external",
    "resource": "raincatcher-cloud",
    "use-resource-role-mappings": true
  }

@austincunningham FYI.

Reply:
Thanks Tom